### PR TITLE
Instituted keeping the previous search and filters/sorts with forms.py & adding logging to views.py

### PR DESCRIPTION
**Pull request by jchristensonCHI**

### CODE REVIEW

File: civiclens/collect/check_doc_status.py
Comment (claireboyd) [inline civiclens/collect/check_doc_status.py:1]: is this used later in the PR or just something that someone else created?

File: civiclens/templates/search_results.html [hunk 1]
```diff
@@ -33,28 +33,20 @@
             <!-- sort section -->
             <span class="h6 font-weight-bold" >Sort by:</span>
             <div class="form-check">
-              <input class="form-check-input" type="radio" name="sort_by" id="most_relevant" value="most_relevant">
```
Comment (claireboyd) [inline civiclens/templates/search_results.html:36]: note about the merge here - the ADA compliance requires titles for inputs, so when you see those pop up in the version on main, keep them but feel free to change the names etc to make them make sense

**Review by claireboyd — APPROVED** [review]
LGTM!

File: civiclens/templates/search_results.html (hunk 1 above)
Comment (jchristensonCHI) [reply civiclens/templates/search_results.html:36]: will do!

Comment (jchristensonCHI) [reply civiclens/templates/search_results.html:36]: Just flagged that my code matches main in slack, and the possibility that an earlier push from myself removed these

**Pull request merged**